### ISSUE TITLE
Scraper Support

### ISSUE THREAD

**Issue by daeks** [daeks/RetroPie-WebGui#7]
- [ ] gamesdb
- [ ] screenscraper
- [ ] google
- [x] individual paths for image, video, marquee, thumbnail

**Comment by grenade44** [comment]
Just rebuilt this and now my images dont show up in the editor.

**Comment by daeks** [comment]
how did you rebuild it? with the retropie module? I imagine that the user is set wrongly. please run your webserver with same user your retropie is running or place a file called "user" in the data directory just containing the name of the user

**Comment by grenade44** [comment]
Yes i readded the .sh file from your rebrand and then installed and then to the retropie:8080 web address and my game lists were there and my images wernt, when they were in the last release.

**Comment by grenade44** [comment]
I just updated and your updates have fixed the problem. thanks
